TMS.Net07.Homework.DaysOfWeek.MiddleLevel/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek.MiddleLevel
+{
+    class Program
+    {
+        static string GetDayOfWeekMiddleLevel(string date)
+        {
+            if (date == "exit")
+            {
+                var exit = "You closed the program!";
+                return exit;
+            }
+            try
+            {
+                DateTime objdate = Convert.ToDateTime(date);
```

Comment:
```suggestion
                DateTime objDate = Convert.ToDateTime(date);
```
In C# we are using CamelCase for variables
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841